### DISCUSSION TITLE
Can't set autoplay:false

### DISCUSSION THREAD

**Discussion by ts-mz** [fancyapps/ui#116] [Q&A] [answered]
Hi, I'm using fancybox 4 for a projekt that mixes images and videos in a lightbox, similarly to what it shown in this showcase:

https://fancyapps.com/playground/NA

I understand from the documentation that the default setting for video is "autoplay". However, I can't seem to be able to switch the autoplay off in the initialization settings. This neither works in the JS playground of the showcase nor in my code, the video will always automatically play. 

`  Video: {
    autoplay: false,
    ratio: 16/9,
  },`

I realize the video may not autoplay for others because of browser policies, but I want to make sure it doesn't autostart.
I suppose that I'm doing something wrong, so I'd appreciate every tip! Thanks!

**Comment by fancyapps** [comment, marked as answer]
Set it like this:

```js
  Html : {
    video : {
      autoplay: false,
      ratio: 4/3,
    }
  },
```

https://fancyapps.com/playground/NB

**Reply by ts-mz** [reply]
Thanks a lot!
